lib/hemp/dependencies.rb

```diff
@@ -1,6 +1,8 @@
 module Hemp
   module Dependencies
     def self.load_files
+      $LOAD_PATH.unshift(File.join RACK_ROOT, "app", "controllers")
```

Comment:
Add parentheses to nested method call `File.join RACK_ROOT, "app", "controllers"`.